### PR TITLE
html-web-page - Junou Lee

### PR DESCRIPTION
**Pull request by junoulee**
image

### CODE REVIEW

File: html-web-page/html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <p>Like what you see? Become a part of the JOTMC family today!</p>
+  <div>
+    <form action="index.html">
+      <div class="your-name">
+        <label for="name"></label>
```
Comment (codingcodymiller) [inline html-web-page/html-web-page/index.html:74]: There is no text labeling this form control element, so no `label` element is necessary

File: html-web-page/html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        <input type="text" name="your-name" id="name" placeholder="Your Name">
+
+        <label for="plan"></label>
```
Comment (codingcodymiller) [inline html-web-page/html-web-page/index.html:77]: There is no text labeling this form control element, so no `label` element is necessary

File: html-web-page/html-web-page/index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        <select name="plan">
+          <option value="Select" id="plan">Select Your Plan</option>
+        </select>
+
+        <label for="submit"></label>
```
Comment (codingcodymiller) [inline html-web-page/html-web-page/index.html:82]: There is no text labeling this form control element, so no `label` element is necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-web-page/html-web-page/index.html (hunk 2 above)
Comment (junoulee) [reply html-web-page/html-web-page/index.html]: image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**